src/main/java/cz/startnet/utils/pgdiff/PgDiffIndexes.java

```diff
@@ -29,7 +29,8 @@
      */
     public static void createIndexes(final PrintWriter writer,
             final PgSchema oldSchema, final PgSchema newSchema,
-            final SearchPathHelper searchPathHelper) {
+            final SearchPathHelper searchPathHelper
+            ) {
```

Comment:
Are you using some sort of automated formatter?  Is this part of the new maven config changes?

Reply:
This is not from maven config. This formatter error was my fault, when I removed the extra parameter that I put.